doc/cephfs/cephfs-shell.rst

```diff
@@ -394,3 +394,20 @@ Usage:
 
 * dir_name - directory under which snapshot should be created or deleted
 
+xattr
+-----
+
+Set or Get extended attribute for a file
+
+Usage :
+
+     xattr [-h] <path> {set|get} <xattr_name> [xattr_value]
```

Comment:
Any particular reason you bundled all setxattr/getxattr commands through this single `xattr` command?

Also, please implement `listxattr`.

Reply:
There is already  `set` and `get` command in cephfs-shell, so I added a single `xattr` command for simplicity.
I have a separate patch for `listxattr` as it requires changes to python bindings.